neps/nep-0509.md

```diff
@@ -57,32 +57,45 @@ As a result, the team sought alternative approaches and concluded that stateless
 * More shards - this is covered in the resharding project.
 * ZK integration.
 * Underlying data structure change (e.g. verkle tree).
-* Change to validator rewards.
 * TBD
 
 ## High level flow
 
-We propose a change to the following parts of the chunk and block production flow:
+The current high-level chunk production flow, if we drop details and edge cases, is as follows:
```

Comment:
just FYI chat gpt suggested the following when I asked to make this sentence more idiomatic: 
```
The current high-level chunk production flow, excluding details and edge cases, is as follows
```